product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md

```diff
@@ -28,20 +38,10 @@ Für die Finalisierung der Epson TSE Imlementierung suchen wir Partner für eine
 
 ### Teilnahme an der Pilotinstallation
 
-Bitte kontakieren Sie [info@fiskaltrust.de](mailto:info@fiskaltrust.de) für die Teilnahme am Epson-TSE-Pilotprojekt, vielen Dank!
+Bitte kontakieren Sie [support@fiskaltrust.de](mailto:support@fiskaltrust.de) für die Teilnahme am Epson-TSE-Pilotprojekt, vielen Dank!
 
 ## Hardware
 
-### Epson TSE, MicroSD
-
-**Artikel Nummer:** 4445-20580
-
-Technische Sicherungseinrichtung (TSE-Modul), Bauform: MicroSD-Karte, Lebensdauer: 20 Mio. Signaturen, Zertifikatslaufzeit: 3 Jahre
-
-Sonderartikel: Storno und Rückgabe unabhängig von der Lieferzeit ausgeschlossen.
-
-
-
 ### Epson TSE, USB
```

Comment:
Should we mention the Printer and TSE Server here

Reply:
agree, but later